### PR TITLE
Fix asyncio NotImplementedError on Win py38

### PR DESCRIPTION
**Pull request by msarahan**
https://stackoverflow.com/questions/58422817/jupyter-notebook-with-python-3-8-notimplementederror

### CODE REVIEW

**Comment by msarahan** [comment]
Somebody take my keyboard away from me!  "patches" in the build section!?  I must be drunk!

File: recipe/0001-set-asyncio-event-loop-policy-on-windows.patch [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+index 0b79f82..99cd01a 100644
+--- a/tornado/platform/asyncio.py
++++ b/tornado/platform/asyncio.py
+@@ -28,6 +28,11 @@ from tornado.ioloop import IOLoop, _Selectable
+ 
+ import asyncio
+ 
++import sys
++
++if sys.platform == 'win32':
```
Comment (mattpap) [inline recipe/0001-set-asyncio-event-loop-policy-on-windows.patch:20]: This fails on Windows and Python 3.6 with:
```
if sys.platform == 'win32':
>       asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
E       AttributeError: module 'asyncio' has no attribute 'WindowsSelectorEventLoopPolicy'
```
A more appropriate version is:
```py
if sys.platform == "win32" and hasattr(asyncio, "WindowsSelectorEventLoopPolicy"):
```
(as used in [tornado](https://github.com/tornadoweb/tornado/blob/f7d94d0e8a57f51ffc0aac5d55ed7bf9f9936b4d/tornado/platform/asyncio.py#L311)).

Comment (jakirkham) [reply recipe/0001-set-asyncio-event-loop-policy-on-windows.patch:20]: Would you like to submit a PR with that patch? 🙂

Comment (mattpap) [reply recipe/0001-set-asyncio-event-loop-policy-on-windows.patch:20]: Sure. Fixed in PR #34.

**Pull request merged**